server/core/src/irods_server_control_plane.cpp

```diff
@@ -198,22 +199,14 @@ namespace irods {
             return irods::error(e);
         }
 
-        std::stringstream pid_str; pid_str << svr_pid;
-        std::vector<std::string> args;
-        args.push_back( pid_str.str() );
+        // Previsouly, we ran kill_pid.py here.
+        // This would send three signals to the process, one right after the other:
+        // SIGSTOP (suspend), SIGTERM (terminate), then SIGKILL (kill)
```

Comment:
but now... not sending SIGSTOP?... can you explain the difference/reasoning from before to now?

Reply:
SIGSTOP would prevent any relevant signal handler from running.

Reply:
yep, okay - just making sure that wasn't an oversight given it was mentioned as being handled prior.